src/interactive-window/interactiveWindowCommandListener.ts

```diff
@@ -348,7 +351,12 @@ export class InteractiveWindowCommandListener implements IDataScienceCommandList
 
     @captureTelemetry(Telemetry.CreateNewInteractive, undefined, false)
     private async createNewInteractiveWindow(connection?: KernelConnectionMetadata): Promise<void> {
-        await this.interactiveWindowProvider.getOrCreate(undefined, connection);
+        try {
+            await this.interactiveWindowProvider.getOrCreate(undefined, connection);
+        } catch (error) {
```

Comment:
Not that this hurts anything, but just interested. Don't errors get logged anyways? Was this fixing some specific case where an error was getting swallowed?

Reply:
whoops, this was just for debugging, didn't mean to leave this in